_extras/customization.md

```diff
@@ -3,18 +3,26 @@ layout: page
 title: Customizing Your Workshop's Website
 permalink: /customization/index.html
 ---
+
+## Table of Content
+
+* TOC
+{:toc}
+
 ## Configuration File `_config.yml`
 
 You should edit the `_config.yml` configuration file in the root directory of your workshop to
 configure some site-wide variables and make the site function correctly:
 
 * `carpentry` - to tell us which carpentry workshop this is, possible values are ("swc" for Software
   Carpentry workshops, "dc" for Data Carpentry workshops, "lc" for Library Carpentry Workshops, or
-  "cp" for general Carpentries events such as instructor trainings).
-* `curriculum` - for Data Carpentry, which one of the curriculum is being taught. Possible values
-  are: `dc-ecology`, `dc-genomics`, `dc-socsci`, `dc-geospatial`.
+  "cp" for general Carpentries events such as instructor trainings for which you should use
+  <https://github.com/carpentries/training-template> for your website template).
+* `curriculum` - for Data Carpentry and Software Carpentry, which one of the curriculum is being
+  taught. Possible values are: `dc-ecology`, `dc-genomics`, `dc-socsci`, `dc-geospatial`,
+  `swc-inflammation` and `swc-gapminder`.
```

Comment:
```suggestion
* `carpentry` - to tell us which carpentry workshop this is. Possible values are:
    - `"swc"` for Software Carpentry workshops,
    - `"dc"` for Data Carpentry workshops,
    - `"lc"` for Library Carpentry workshops, and
    - `"cp"` for general Carpentries events such as instructor trainings (for which you should use
      <https://github.com/carpentries/training-template> as the website template)
* `curriculum` - to tell us which curriculum is being taught.
  At the moment, applicable to Software and Data Carpentry workshops only.
  Possible values are:
    - `"dc-ecology"`, `"dc-genomics"`, `"dc-socsci"`, and `"dc-geospatial"` for Data Carpentry workshops
    - `"swc-inflammation"` and `"swc-gapminder"` for Software Carpentry workshops
```

Reply:
I suggest using nested lists to show possible values. I also suggest adding double quotation marks around possible values to minimize possible confusion because things like `curriculum` and `carpentry` in `_config.yml` don't need those and possible values do.